### DISCUSSION TITLE
Dynamics of an object

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#24343] [Q&A Modules: Solid mechanics] [answered]
Is there an example code that models the the Newtonian motion of an object where there can be an initial condition that accelerates the object, and the following trajectory of the object is influenced by gravity and the initial condition?

**Comment by GiudGiud** [comment]
@jiangwen84 @recuero

**Reply by GiudGiud** [reply]
@Strawberrybrother answer might matter to you too

**Reply by recuero** [reply, marked as answer]
I cannot find such an example now. But I have played with a similar setup using the Newmark integrator. You can using gravity and initial conditions on the variables (including auxiliary velocity and acceleration) to achieve what you describe. Note that you shouldn't use mass damping if you want to get physical results, as it will damp out rigid body motion.
